rtengine/improccoordinator.cc

```diff
@@ -32,6 +32,24 @@
 #ifdef _OPENMP
 #include <omp.h>
 #endif
+
+namespace
+{
+
+using namespace rtengine;
+Coord2D translateCoord(ImProcFunctions& ipf, int fw, int fh, int x, int y) {
+    const std::vector<Coord2D> points = {Coord2D(x, y)};
+
+    std::vector<Coord2D> red;
+    std::vector<Coord2D> green;
+    std::vector<Coord2D> blue;
+    ipf.transCoord(fw, fh, points, red, green, blue);
```

Comment:
That makes me wonder why `ImProcFunctions::transCoord()` isn't `const`... :thinking:

Reply:
You mean that, if `transCoord` were const, i could have defined the ipf param as const ?
If i try to define `transCoord` as const, the compiler barks at me, because it's calling a bunch of non-const functions internally (if i understand correctly)

Reply:
Yep. Wait a minute, I'll push a fix to `dev`...

Reply:
Done. Now you can pass it as `const&`. :smiley:

Reply:
Cool! Done.